### DISCUSSION TITLE
更新VX3.50.0.33无法使用

### DISCUSSION THREAD

**Discussion by Aixinstodio** [huiyadanli/RevokeMsgPatcher#276] [General]
![2022-01-15_19-20-34](https://user-images.githubusercontent.com/50140524/149619897-b5015c22-6ab6-43a0-b9bb-c3ae88b0b390.png)

**Comment by xuwenkai357** [comment]
收到了

**Comment by huiyadanli** [comment]
#275 已经支持了

**Reply by Aixinstodio** [reply]
辛苦了 谢谢
